### PR TITLE
Created AWS Example Projects With HTTP and MQTT

### PR DESCRIPTION
**Pull request by dabdoue**
2 examples projects, one for connecting to AWS via MQTT, another for HTTP, as well as a README that describes how to setup AWS environment to work with Moddable Project

### CODE REVIEW

File: examples/network/aws/http-aws/main.js
Comment (phoddie) [inline examples/network/aws/http-aws/main.js:1]: Copyright year should be 2023 on all source files

File: examples/network/aws/moddable-aws-guide.md
Comment (phoddie) [inline examples/network/aws/moddable-aws-guide.md:1]: In our documentation, we usually include a copyright notice and a revision date. The revision date an be helpful for readers. 

```
Copyright 2023 Moddable Tech, Inc.<BR>
Revised: March 27, 2023
```

File: examples/network/aws/mqtt-aws/manifest.json [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+	"include": [
+		"$(MODDABLE)/examples/manifest_base.json",
+		"$(MODDABLE)/examples/manifest_net.json",
+		"$(MODDABLE)/modules/network/mqtt/manifest.json",
+		"$(MODDABLE)/modules/crypt/tls.json"
+	],
+	"config": {
+		"ssid": "ModdableWifi",
+		"password": "12345679"
+	},
```
Comment (phoddie) [inline examples/network/aws/mqtt-aws/manifest.json]: We usually avoid putting credentials, even as an example, in the example manifests. That helps avoid credentials being accidentally committed to a repository. Instead, we encourage developers to pass credentials on the command line.

File: examples/network/aws/http-aws/manifest.json [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+			"./main",
+			"$(MODULES)/network/http/*"
+		]
+	},
+	"preload": [
+		"http"
+	],
+	"resources": {
+		"*": [
+            "./CA1-key",
```
Comment (phoddie) [inline examples/network/aws/http-aws/manifest.json]: This line appears to be using spaces for indenting while all other lines use tabs.

**Review by phoddie — CHANGES_REQUESTED** [review]
This looks really good. I noted a few minor items to clean-up. Once you take care of those, we can get this merged. Thanks!

**Comment by PatrykStachura** [comment]
It is not working as expected. Connection is achived, pipe also is not broken at the end, the AWS IoT registering connection, but device is not reactive. It have no option to publish/subscribe date, so i believe there's something wrong with secureCerts properties of constructor of client

**Comment by PatrykStachura** [comment]
Why you close that brother?

**Comment by dabdoue** [comment]
@PatrykStachura , thanks for the report. We've confirmed that the examples do work when the AWS endpoint is configured correctly. Let's try to sort out what's different about your set-up.

Which example are you working with, http or mqtt? 

Also, please ensure you followed Amazon's docs [here](https://docs.aws.amazon.com/iot/latest/developerguide/create-iot-resources.html) and have a policy attached to your AWS Thing that allows for publishing, subscribing, and receiving. 

Additionally, which Root CA certificate are you using, RSA or ECC (CA1 or CA3)?

If you could please make this an issue instead, including answers to these questions, that would be appreciated so that others can see this if they have the same issues as you.

Thanks!

**Comment by phoddie** [comment]
The PR has been merged. Once a PR Is merged, we close it.

We're happy to help with the issue you describe. Please open an issue for that. Thank you.

**Comment by PatrykStachura** [comment]
@dabdoue & @phoddie thank you for your responses. The example implemented by me using mqtt protocol, i convert certificates as it is in readme and use an `Reasource` class to provide each cert to config properties. Regarding CA certificate i'm currently using ` RSA 2048 bit key: Amazon Root CA 1 `. I will provide it an issues

**Pull request closed without merging**